### PR TITLE
Added script that copies cromwell import output to a single location

### PR DESCRIPTION
**Pull request by Fedja**
where deployment script can pick it up easily

### CODE REVIEW

**Review by juhis — COMMENTED** [review]
One of the best things I've seen in a while. I'd add optional output_nodes - e.g. for pheweb import we actually don't need to run the annotation task every time (it's completely separate from the results and basically gives just rsids and genes) so it would be flexible if annotation outputs could be optional so that this works with or without the annotation task. Not totally necessary though as we can also just remove the annotation output_nodes

**Comment by Fedja** [comment]
Added switch to skip annotation nodes

File: scripts/copy_cromwell_import_to_bucket_puddle.py [hunk 1]
```diff
@@ -38,6 +46,10 @@ def run():
         raise Exception(f'Error requesting metadata. Cromwell message: {ret["message"]}')
 
     all_files = {}
+
+    if not args.skip_annotation:
+        output_nodes.extend(annot_nodes)
```
Comment (juhis) [inline scripts/copy_cromwell_import_to_bucket_puddle.py:51]: output_nodes variable doesn't exist

File: scripts/copy_cromwell_import_to_bucket_puddle.py [hunk 2]
```diff
@@ -7,13 +7,21 @@
...
+
+annot_nodes = ["pheweb_import.annotation.bed", "pheweb_import.annotation.gene_trie",
+    "pheweb_import.annotation.sites","pheweb_import.annotation.trie1","pheweb_import.annotation.trie2"]
 
 def run():
     parser = argparse.ArgumentParser(description="Run x-way meta-analysis")
     parser.add_argument('cromwell_hash', action='store', type=str, help='Configuration file ')
     parser.add_argument('destination_bucket', action='store', type=str, help='Result file')
 
+    parser.add_argument('--skip_annotation', action='store', type=str, help='Result file')
```
Comment (juhis) [inline scripts/copy_cromwell_import_to_bucket_puddle.py:23]: shouldn't this be store_true

File: scripts/copy_cromwell_import_to_bucket_puddle.py [hunk 3]
```diff
@@ -7,13 +7,21 @@
...
+    "pheweb_import.matrix.pheno_gene","pheweb_import.matrix.phenolist",
+    "pheweb_import.matrix.top_hits_1k","pheweb_import.matrix.top_hits_json","pheweb_import.matrix.top_hits_tsv",
+    "pheweb_import.pheno.manhattan","pheweb_import.pheno.pheno_gz","pheweb_import.pheno.pheno_tbi","pheweb_import.pheno.qq"]
+
+annot_nodes = ["pheweb_import.annotation.bed", "pheweb_import.annotation.gene_trie",
+    "pheweb_import.annotation.sites","pheweb_import.annotation.trie1","pheweb_import.annotation.trie2"]
 
 def run():
     parser = argparse.ArgumentParser(description="Run x-way meta-analysis")
     parser.add_argument('cromwell_hash', action='store', type=str, help='Configuration file ')
```
Comment (juhis) [inline scripts/copy_cromwell_import_to_bucket_puddle.py:20]: help texts of most arguments

**Review by juhis — CHANGES_REQUESTED** [review]

**Review by juhis — APPROVED** [review]
Beautiful:)

**Pull request merged**